### PR TITLE
Update Integration Rev

### PR DESCRIPTION
**Pull request by OjesviPandita**
Revised version of Integration steps

### CODE REVIEW

File: docs/integrate.md [hunk 1]
```diff
@@ -31,40 +31,105 @@ Ultra launches the webpage inside a mobile’s web-view. Here, you may pick your
...
 
+!!!note
+    The logic presented here works in both React Native and HTML/PWA. These methods are asynchronous and always return a [promise](https://javascript.info/promise-basics) that gets resolved with some values except few other JavaScript methods that have no return value.
+
 ## 3: Ultra User Login Process
 In this section, you learn how to fetch user details securely from Flipkart’s domain. A user can gain entry into your application only after allowing permissions to share their information (such as name, email, phone number etc.). This step guides you about the generation of grant token, fetching user’s information and login the user into your application.
 
 ### Generate grant token
-You might have come across the terms which are very common in the technical world of authorization (OAuth) world such as “Granted Scopes” and “Grant tokens”. To know more, read [this](https://alexbilbie.com/guide-to-oauth-2-grants/).
+You might have come across the terms which are very common in the technical world of authorization (OAuth) world such as `Granted Scopes` and `Grant tokens`. To know more, [read this](https://alexbilbie.com/guide-to-oauth-2-grants/).
```
Comment (MSReehal) [inline docs/integrate.md]: The word "world" is used twice in this sentence:
"... in the technical **world** of authorization (OAuth) **world** ..."

File: docs/integrate.md [hunk 2]
```diff
@@ -31,40 +31,105 @@ Ultra launches the webpage inside a mobile’s web-view. Here, you may pick your
...
+!!!note
+    It is mandatory to redirect via `fapp://` to the Ultra container.
+    You should not open any UI page of your application outside Ultra container.
+
+> Your money is transferred to an MID account which has to be configured. [Contact our team](contact.md) to onboard your MID account with us before starting the integration since it takes a significant amount of time.
+
+In case of refunds, integrate our [refund API](backend.md#refund) with your application as per your business needs.
+
+## 5: OMS Integration
+After the order has been placed and the payment has been successfully completed, the user would want to know the status of the placed order. So, now it is the time to call the [`Flipkart OMS APIs`](oms.md) to know about the order status.
```
Comment (MSReehal) [inline docs/integrate.md]: Please change "...to know about the order status." to something like "...to update it with the status of the order in your application."

File: docs/integrate.md (hunk 1 above)
Comment (OjesviPandita) [reply docs/integrate.md]: Done. Please check the latest commit.

File: docs/integrate.md (hunk 2 above)
Comment (OjesviPandita) [reply docs/integrate.md]: Done. Please check the latest commit.

**Pull request merged**